tests/e2e/PHPUnit11/README.md

```diff
@@ -0,0 +1,9 @@
+# Title
+
+* Link to github ticket if relevant
```

Comment:
Please insert a link to the original issue/pr and please update summary below explaining what issue is covered by e2e test